mod_ci/controllers.py

```diff
@@ -369,22 +369,25 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
 
         test_branch.checkout(True)
 
+        pull = repository.pulls('{pr_nr}'.format(pr_nr=test.pr_nr)).get()
+        if pull['mergeable'] == 'false':
```

Comment:
There's no check if that "pull" variable contains anything at all. What if an error happens?